src/app/item-popup/PlugTooltip.tsx

```diff
@@ -27,15 +27,26 @@ export default function PlugTooltip({
 }) {
   // TODO: show insertion costs
 
+  // display perk's synergy with masterwork stat
+  const synergyStat =
+    item.masterworkInfo &&
+    plug.plugItem.investmentStats &&
+    item.masterworkInfo.statHash &&
+    plug.plugItem.investmentStats.some((stat) => {
+      return (
```

Comment:
No need to have braces and a return here, you can just have a compact arrow function.

Reply:
:+1: